ush/build_all_cmake.sh

```diff
@@ -80,7 +80,7 @@ else
 fi
 
 if [ $build_type = PRODUCTION -o $build_type = DEBUG ] ; then
-  cmake -DBUILD_UTIL=ON -DBUILD_NCDIAG_SERIAL=ON -DCMAKE_BUILD_TYPE=$build_type -DBUILD_CORELIBS=OFF ..
+  cmake -DBUILD_UTIL=ON -DMPI3FLAG=-DMPI3 -DMPI3=ON -DBUILD_NCDIAG_SERIAL=ON -DCMAKE_BUILD_TYPE=$build_type -DBUILD_CORELIBS=OFF -DENKF_MODE=WRF ..
```

Comment:
Don't change the default build mode to be WRF for GSI master.

Reply:
The GSI master code can't compiled in debug mode on Cray without this change.

Reply:
I'm referring to the ENKF_MODE WRF part...

Reply:
Sorry, misunderstood your comments. Removed the -DENKF_MODE=WRF.